api/src/main/java/org/pmiops/workbench/dataset/DataSetServiceImpl.java

```diff
@@ -815,6 +815,58 @@ private String getQualifiedColumnName(Domain currentDomain, String columnName) {
             + "head results.P2.assoc");
   }
 
+  @Override
+  public List<String> generateHailDemoCode(String qualifier) {
+    final String phenotypeFilename = "phenotypes_annotations_" + qualifier + ".txt";
```

Comment:
.tsv?

Reply:
The Hail tutorial had it as ".txt" but tsv makes sense.